modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3312,6 +3312,7 @@ void ONNXImporter::parseQuantDequant(LayerParams& layerParams, const opencv_onnx
     CV_Assert(node_proto.input_size() == 2 || node_proto.input_size() == 3);
     layerParams.type = (node_proto.op_type() == "QuantizeLinear") ? "Quantize" : "Dequantize";
     int axis = layerParams.get<int>("axis", 1);
+    int block_size = layerParams.get<int>("block_size", 0);
```

Comment:
Attributes in a node_proto is parsed already (populateNet() -> handleNode() -> [getLayerParams()](https://github.com/opencv/opencv/blob/f1aef892bfe7b30255050275fdb80d2ec108a9a4/modules/dnn/src/onnx/onnx_importer.cpp#L505)) and is not necessary to do it manually if you are not making any modification to the value.